extensions/observability-devservices/common/src/main/java/io/quarkus/observability/common/config/LgtmConfig.java

```diff
@@ -9,15 +9,28 @@
 
 @ConfigGroup
 public interface LgtmConfig extends GrafanaConfig {
+
+    /**
+     * The name of the Grafana LGTM Docker image.
+     */
     @WithDefault(ContainerConstants.LGTM)
     String imageName();
 
+    /**
+     * The Docker network aliases.
+     */
     @WithDefault("lgtm,lgtm.testcontainer.docker")
     Optional<Set<String>> networkAliases();
 
+    /**
+     * The label of the container.
+     */
     @WithDefault("quarkus-dev-service-lgtm")
     String label();
 
+    /**
+     * The port on which the LGTM container will be exposed.
```

Comment:
... LGTM's OTLP port will be exposed ...

Reply:
Fixed!